### ISSUE TITLE
Extracting Latent Spaces

### ISSUE THREAD

**Issue by 42elenz** [lucidrains/tab-transformer-pytorch#24]
Dear Team,
first thank you for your awesome work. This is less an issue than a question.

I have the goal to train a model with a contrasting-learning-method between tabular data and MRIs. Is it possible to get a latent space representation from your data that I could use to compute it with my images?
As I understand just the categorial variables go through the transformer and the continues through a Layer-Normalization. 
As your model is working on labeled data as I understand, does this latent space actually has any specific meeting?

What could I return from your code to use it as latent space? And would it be possible to generate data back from this latent space in human readable data? 

So to summarize:
I want a usable latent space from tabular data that represents the relationships between the items in a meaningful way to use contrastive learning on it. Do you think your TabTransformer is suitable for this?

Thank you very much for your work and I hope you can help me.
